### DISCUSSION TITLE
Hardware Q - Why two USB type C?

### DISCUSSION THREAD

**Discussion by Presjar** [mathoudebine/turing-smart-screen-python#11] [Q&A] [answered]
![Screenshot_20220429-220924_AliExpress](https://user-images.githubusercontent.com/18156203/165947023-4de0fbf5-8963-4efe-a1f9-2261c8684410.png)
I found on the sellers website some reviews showing the rear of the PCB. As I have not purchased the device yet, I'm wondering why there is two USB type C connectors?

**Comment by mathoudebine** [comment]
Hi, this is a good question! The port near the side is the one to be used for connecting the display to a computer.
For the port in the center, i could not try it because the port is soldered to the screen without any space between connector and rear panel, so any standard USB-C cable cannot be connected to it because of the plastic around the connector (here in red):
![image](https://user-images.githubusercontent.com/38615348/165948745-92c9e577-1dea-492e-9466-5159031e6d31.png)
Only these kind of "thin" cables could work:
![image](https://user-images.githubusercontent.com/38615348/165949403-0aa8c7ab-f941-4ef9-a2f1-b0283f29d2e6.png)
No info from the seller, so my guess is that the connector is used to flash the firmware in factory.

**Reply by Presjar** [reply]
![Screenshot_20220429-232120_AliExpress](https://user-images.githubusercontent.com/18156203/165952649-2595831f-5f4a-4391-929e-866c8141cd63.png)
Interesting! Strange to have to stick the cable out the top of the screen. Keeping it looking neat when mounting inside a PC against glass would be a bit of a pain. Maybe I would need a 90 / 180 deg cable.

Also those type c connectors are not low cost, when using 100s or 1000s, so using two on the PCB seems a waste when it is never used by the customer.

**Reply by Presjar** [reply]
Could also get this for $0.75 and see what the second connector actually does haha.
![1598100573-546534408](https://user-images.githubusercontent.com/18156203/165953789-df700db3-5d10-418b-9260-abe60eaec1f4.jpg)

**Reply by mathoudebine** [reply]
Absolutely! Or use a cutter and plane a spare USB-C cable if you have one 😉 
Maybe both ports can be used to communicate with the display

**Comment by fr-fhe** [comment, marked as answer]
On my Turing Smart Screen, i can confirm the two USB ports are the same.
I think this is a way to have the cable in one direction or another (depending screen flip horizontally or vertically).

**Comment by Presjar** [comment]
Interesting! Thank you. I had not got around to purchasing yet.
